### DISCUSSION TITLE
Angular + Vercel + Spring boot: Getting error 405 when calling API

### DISCUSSION THREAD

**Discussion by annvbitnemo** [vercel/community#3569] [Help]
### Summary

I'm looking to deploy my Angular application to Vercel to connect to my server (written in springboot). On local, everything is fine. But after deploying, the login api (with POST method) always returns 405 error and on the server backend log there is no response. I also use proxy file for my application.

### Steps to Reproduce

Here is proxy.conf.json
`{
  "/api": {
    "target": "https://mydomain.com:8081/",
    "secure": false,
    "logLevel": "debug",
    "changeOrigin": true
  }
}`

**Comment by Kadouani** [comment]
Hello,

I have the same error did you resolve your problem please?
